event/events.go

```diff
@@ -140,7 +140,7 @@ type Unsigned struct {
 	RedactedBecause *Event          `json:"redacted_because,omitempty"`
 	InviteRoomState []StrippedState `json:"invite_room_state,omitempty"`
 
-	HungryRowID int64 `json:"com.beeper.hungry_row_id,omitempty"`
+	BeeperHsOrder int64 `json:"com.beeper.hs.order,omitempty"`
```

Comment:
```suggestion
	BeeperHSOrder int64 `json:"com.beeper.hs.order,omitempty"`
```

(could also do `BeeperHomeserverOrder`, but if it's an initialism it has to be in consistent case)

Reply:
We seem to call it "HS" ordering on Synapse as well.